.github/workflows/bridge-ui-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Bridge UI to Vercel
```

Comment:
is this a preview or production deployment?
is this a testnet (which one) or mainnet deployment?

Reply:
It's meant to be generic, so it can be called/used by any testnet or the mainnet.